### ISSUE TITLE
Request clarification on `flutter/tests` definitions

### ISSUE THREAD

**Issue by jtmcdole** [flutter/flutter#155170]
> You will also need to land your change in two parts, so that people have time to migrate

How much time is afforded to migrate?

> the people listed as contacts for the test are not responsive (within 72 hours).

Define `responsive`. Is this an `ack` to recognize the request. It is an acceptance of bug?

In my opinion, we should have crisper definitions and timelines so flutter and customer engineers aren't guessing.

**Comment by darshankawar** [comment]
Adding `need additional triage` since this seems to be related to `flutter/tests` repo which is outside of flutter repo, so I am not sure which team should this belong to.

**Comment by jtmcdole** [comment]
I'm assuming its Framework.

**Comment by srawlins** [comment]
Agree, the current text only mentions the 72 hours once, and isn't too explicit about how it should be done. I mailed a PR to a customer test repo that was required to unblock a Dart SDK change. The owner noted that they don't get notifications of all PRs, and requested that I @mention them any such PRs. I cannot keep track of any special process between me and this one other contributor, but I think such a policy might be good in general.

It would probably be good to be explicit and forthright in any such PRs. Some text like "This PR is required for a change to Dart / Flutter, and as this repo has formal Flutter customer tests, this PR is required unblock the Dart / Flutter work. @mention @mention. PTAL within 72 hours."

TL;DR mention something about @mentioning owners in PRs to customer repos.

CC @matthew-carroll

**Comment by matthew-carroll** [comment]
There's also an email address listed for contact. To my knowledge, no one has ever emailed me at the posted contact email for the tests I placed in the register.
